From 7a77c2a39d5efa32dfb430192cf4153d80fc0f97 Mon Sep 17 00:00:00 2001
From: Rex Lu <rex.lu@mediatek.com>
Date: Wed, 18 Dec 2024 13:31:12 +0800
Subject: [PATCH] refactor detect mlo support

Signed-off-by: Rex Lu <rex.lu@mediatek.com>
---
 src/nl80211/wld_nl80211_parser.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/nl80211/wld_nl80211_parser.c b/src/nl80211/wld_nl80211_parser.c
index 44e586a..2efd7b6 100644
--- a/src/nl80211/wld_nl80211_parser.c
+++ b/src/nl80211/wld_nl80211_parser.c
@@ -1026,7 +1026,9 @@ swl_rc_ne wld_nl80211_parseWiphyInfo(struct nlattr* tb[], wld_nl80211_wiphyInfo_
     }
     NLA_GET_VAL(pWiphy->nStaMax, nla_get_u32, tb[NL80211_ATTR_MAX_AP_ASSOC_STA]);
     /* check MLO support */
-    pWiphy->suppMlo = (tb[NL80211_ATTR_MLO_SUPPORT] != NULL);
+    if(tb[NL80211_ATTR_MLO_SUPPORT])
+        pWiphy->suppMlo = true;
+
     s_parseIfTypes(tb, pWiphy);
     s_parseIfCombi(tb, pWiphy);
     s_parseWiphyBands(tb, pWiphy);
-- 
2.45.2
-- 
